src/core/packages/notifications/browser/index.ts

```diff
@@ -14,6 +14,7 @@ export type {
   ToastInput,
   IToasts,
   ToastInputFields,
+  NotificationCoordinator,
```

Comment:
AFAIK, we need to export `NotificationCoordinatorPublicApi` and `NotificationCoordinatorState`, since `NotificationCoordinator` depends on them.